foundation/foundation-archetype/src/main/resources/archetype-resources/__rootArtifactId__-docker/pom.xml

```diff
@@ -85,7 +84,6 @@
                             <groupId>${group.fabric8.plugin}</groupId>
                             <artifactId>docker-maven-plugin</artifactId>
                             <configuration>
-                                <skip>false</skip>
                                 <images>
                                     <image>
                                         <name>${docker.project.repository.url}${project.artifactId}:${project.version}</name>
```

Comment:
S: We take this opportunity to also align the name between `dev` and `ci`. On the baseline we stopped trimming off the registry URL locally a long time ago and it's generally made development much easier/smoother.

Reply:
If we align the name, then I think we can kill of the `ci` profile altogether.

Reply:
Already had the code ready but running into some farbic8 docker build issue while testing, which will require migration script for downstream project so will separate out to a different ticket